source/pdo_sqlsrv/pdo_stmt.cpp

```diff
@@ -427,7 +427,7 @@ int pdo_sqlsrv_stmt_describe_col(pdo_stmt_t *stmt, int colno TSRMLS_DC)
 #else
     pdo_sqlsrv_stmt* driver_stmtt = reinterpret_cast<pdo_sqlsrv_stmt*>( stmt->driver_data );
     driver_stmtt->set_func( __FUNCTION__ );
-    int length = strlen(__FUNCTION__);
+    int length = strlen(__FUNCTION__)+strlen(": entering");
```

Comment:
Using the macro `PDO_LOG_STMT_ENTRY` instead of the `#ifndef - #else -#endif` block from line   [425](https://github.com/Microsoft/msphpsql/pull/157/files#diff-f4508b9b04d04cf713bdf690984d02aeR425)-[433](https://github.com/Microsoft/msphpsql/pull/157/files#diff-f4508b9b04d04cf713bdf690984d02aeR433) should also do the trick.
This is duplicated code from line [337](https://github.com/Microsoft/msphpsql/pull/157/files#diff-f4508b9b04d04cf713bdf690984d02aeR337) (`PDO_LOG_STMT_ENTRY` ).